webapp/src/main/resources/index.html

```diff
@@ -0,0 +1,45 @@
+<!doctype html>
+<html lang="en" xmlns="http://www.w3.org/1999/html">
+    <head>
+        <meta charset="UTF-8">
```

Comment:
Fun fact this is one of my biggest pet peeves (not your fault, I realize it's the spec). How the hell is a text parser supposed to find `<meta charset="UTF-8">` if it wasn't already using the correct encoding? And why doesn't HTML close all tags `/>` 😭

Reply:
Haha. The web is a crazy-wild-beast.